main.js

```diff
@@ -0,0 +1,35 @@
+var swiper = new Swiper('.swiper--preview-img', {
+  loop: true,
+  spaceBetween: 10,
+  slidesPerView: 4,
+  freeMode: true,
+  watchSlidesProgress: true,
+
+});
+
+var swiper = new Swiper('.swiper--view-img', {
+  loop: true,
+  spaceBetween: 10,
+  navigation: {
+    nextEl: ".swiper-button-next",
+    prevEl: ".swiper-button-prev",
```

Comment:
- Mismo aviso, preferible usar clases `js-`
- Usa un solo tipo de comillas, en lineas anteriores usaste sencillas y ahora tienes dobles. El cambio deberia hacerse solo cuando necesites usar en el string la misma que estas usando en el resto del codigo. Por ej, si usas sencilla y necesitas un string que use una sencilla `const x = "'"`
```suggestion
    nextEl: '.js-swiper-button-next',
    prevEl: '.js-swiper-button-prev',
```